### PR TITLE
Fixes #24377: Migrate the condition_from_command method to logger_v4

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/24377

### CODE REVIEW

**Review by amousset — APPROVED** [review]

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

File: tests/acceptance/30_generic_methods/file_from_local_source_with_check.cf [hunk 1]
```diff
@@ -47,40 +65,48 @@ bundle agent init
...
-    "content_1" expression => strcmp("${content_1}", "test file");
-    "content_3" expression => strcmp("${content_3}", "test file");
-    "content_4" expression => strcmp("${content_4}", "test file");
-
-    "ok_1" expression => "file_1.content_1.file_from_local_source_with_check_${init.destination_file_canon_1}_repaired.!file_from_local_source_with_check_${init.destination_file_canon_1}_error";
-    "ok_2" expression => "!file_2.file_from_local_source_with_check_${init.destination_file_canon_2}_error";
-    "ok_3" expression => "file_3.content_3.file_from_local_source_with_check_${init.destination_file_canon_3}_kept";
-    "ok_4" expression => "file_4.content_4.file_from_local_source_with_check_${init.destination_file_canon_4}_repaired.!file_from_local_source_with_check_${init.destination_file_canon_4}_error";
-    "ok" expression => "ok_1.ok_2.ok_3.ok_4";
+    "classes_ok"   expression => "ph1_ok.ph2_ok.ph3_ok.ph4_ok";
```
Comment (amousset) [inline tests/acceptance/30_generic_methods/file_from_local_source_with_check.cf:96]: Out of curiosity, do you know where the `ph` word comes from?

**Review by amousset — APPROVED** [review]

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**